Makefile

```diff
@@ -397,6 +397,11 @@ test-e2e-source:
 test-e2e-images:
 	ginkgo $(GINKGO_FLAGS) -focus="odo supported images e2e tests" tests/e2escenarios/
 
+# Run devfile e2e tests: odo devfile supported tests
+.PHONY: test-e2e-devfile
+test-e2e-devfile:
```

Comment:
Please add this make target to travis https://github.com/openshift/odo/blob/master/.travis.yml#L147-L159

Reply:
done